_inc/client/components/connect-button/style.scss

```diff
@@ -9,7 +9,7 @@
 	border-color: $green-secondary;
 	color: $white;
 
-	&:hover, &:focus {
+	&:hover:not([disabled]), &:focus:not([disabled]) {
```

Comment:
Does the `:disabled` selector work for what you need?

Reply:
The `disabled` color is supposed to be the same as the existing button. The non-disabled state is where we want to make the difference. Targeting the :disabled state and removing `not([disabled])` would replace one rule with two and would not automatically inherit the disabled state color from the base CSS.

Reply:
For this comment thread, I meant `not(:disabled)`.

---

> The `disabled` color is supposed to be the same as the existing button.

Then how does the user know it's disabled?